ui/app/components/pending-tx.js

```diff
@@ -298,7 +310,7 @@ PendingTx.prototype.render = function () {
             type: 'submit',
             value: 'ACCEPT',
             style: { marginLeft: '10px' },
-            disabled: insufficientBalance || !this.state.valid,
+            disabled: insufficientBalance || !this.state.valid || !isValidAddress,
```

Comment:
`this.state.valid` and `isValidAddress`
are we being redundant here? maybe not but just wanted to check